src/App.tsx

```diff
@@ -2,15 +2,23 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { useState } from 'react';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState<Movie[]>(moviesFromServer);
+
+  const addMovie = (newMovie: Movie) => {
+    setMovies([...movies, newMovie]); // Добавляем новый фильм в начало списка
```

Comment:
```suggestion
    setMovies([...movies, newMovie]);
```
Not critical, but important to understand generally. No need to add a lot of comments - it's clear from the code what is going on here. And you should try to make your code self-explanatory (with meaningful variable names, easy-to-understand structure, etc). Redundant comments only add "noise" to the code. So remove such comments everywhere 🙂 

Comments required in places with complex logic, to explain some not tipical approaches, etc.